.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  env: {
+    es2021: true,
```

Comment:
@jerome-benoit What do you think about this flag? It was added by the init script :thinking:

Reply:
Hum, is it possible to enable the same es version than the one in tsconfig.json?

Reply:
`Environment key "es2018" is unknown` ... no

Reply:
Let's suppose the default configured values are sensible ones and keep them ;)